index.html

```diff
@@ -57,6 +57,266 @@ <h1 id="primary-text">
+                <div class="bottom-area">
+                    <div class="top">
+                        <p class="text">
+                            Multi-post stories<br>
+                            Gain+Glory
+                        </p>
+                    </div>
+                    <div class="tech">
+                        <div class="tech-card">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
+                    <div class="bottom">
+                        <div class="see-btn">
+                            <p>See Project</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="top-area">
+        
+                </div>
+                <div class="bottom-area">
+                    <div class="top">
+                        <p class="text">
+                            Multi-post stories<br>
+                            Gain+Glory
+                        </p>
+                    </div>
+                    <div class="tech">
+                        <div class="tech-card">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
+                    <div class="bottom">
+                        <div class="see-btn">
+                            <p>See Project</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        
+        <section id="about-me">
+            <div id="ab-me-img"></div>
+            <div id="ab-me-1-img"></div>
+            <div id="ab-me-desktop-img"></div>
+            <div id="ab-me-desktop-1-img"></div>
+            
+            <p class="title">About me</p>
+            <p class="text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+            </p>
+            <div class="see-btn">
+                <p class=".text">
+                    Get my resume
+                </p>
+            </div>
```

Comment:
- [ ] As small as this place is, it still manages to get an overflow within itself :smile: Kindly remove the overflow by using relative units on your height values so that that scroll bar can be removed in order to match design :wink: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/243130310-8599b195-2d98-4111-98f8-82cb6a268431.png)|![image](https://user-images.githubusercontent.com/99715640/243130328-d4ee7315-c726-4ea8-b42e-7aa8a192c48b.png)|